http/handlers.go

```diff
@@ -146,19 +145,19 @@ func ErrorHandlerNoLog() func(context.Context, error, http.ResponseWriter) {
 }
 
 // ErrorHandler encodes the reply when there is an error.
-func ErrorHandler(logger cs.Logger) func(context.Context, error, http.ResponseWriter) {
+func ErrorHandler(logger log.Logger) func(context.Context, error, http.ResponseWriter) {
 	return func(_ context.Context, err error, w http.ResponseWriter) {
 		switch e := errors.Cause(err).(type) {
 		case security.ForbiddenError:
-			logger.Log("ErrorHandler", http.StatusForbidden, "msg", e.Error())
+			logger.Info("ErrorHandler", http.StatusForbidden, "msg", e.Error())
 			w.WriteHeader(http.StatusForbidden)
 		case Error:
-			logger.Log("ErrorHandler", e.Status, "msg", e.Error())
+			logger.Error("ErrorHandler", e.Status, "msg", e.Error())
```

Comment:
This is an error explicitly created by our components. Shall we log this as Info or Error ?

Reply:
After discussion, we don't want to log at all at this stage. All errors explicitly created or wrapped by our component would have already been logged. 
We keep logging in the default case only, it should never happen expect for rate limiting and errors we forgot to managed correctly